### PR TITLE
11 p127 tiu template get defaults

### PR DESCRIPTION
**Pull request by sidjtd**
Finally! Completed and first usage of getList parameterService for me out of my RPC's.
No issues rebasing but had to --force origin due to diverging.

### CODE REVIEW

File: prototypes/parameters/p127-spec.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ *
+ * This test suite demonstrates the invocation of various VistA RPCs and their
+ * expected outputs.
+ */
+
+// binding and driver for the GT.M language and database
+const nodem = require('nodem');
+
+// utility library
+// const _ = require('underscore');
```
Comment (EdisonTing) [inline prototypes/parameters/p127-spec.js]: minor thing. Remove unused comments/codes.

Comment (sidjtd) [inline prototypes/parameters/p127-spec.js]: You got it!

File: prototypes/parameters/p127L-spec.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env node
+
+'use strict';
+
+// binding and driver for the GT.M language and database
+const nodem = require('nodem');
+// utility library
+// const _ = require('underscore');
```
Comment (EdisonTing) [inline prototypes/parameters/p127L-spec.js]: same here.

File: prototypes/parameters/p127_tiuTemplateGetDefaults.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+'use strict';
+
+// const _ = require('underscore');
+// const moment = require('moment');
+// const Handlebars = require('handlebars');
+
```
Comment (EdisonTing) [inline prototypes/parameters/p127_tiuTemplateGetDefaults.js]: same here.

**Pull request merged**